common/shlibs

```diff
@@ -865,6 +865,7 @@ libquadmath.so.0 libquadmath-4.4.0_1
 libgfortran.so.5 libgfortran-8.2.0_1
 libgnarl-12.so libada-12.2.0_1
 libgnat-12.so libada-12.2.0_1
+libsharpyuv.so.0 libwebp-1.3.0_1
```

Comment:
libsharpyuv is a separate library from libwebp that can be used independently (libwebp itself even depends on libsharpyuv). In light of that, should I split libsharpyuv into a separate subpackage?

Reply:
Sounds reasonable, yes.